network/bridge_endpointclient_linux.go

```diff
@@ -81,6 +83,26 @@ func (client *LinuxBridgeEndpointClient) AddEndpointRules(epInfo *EndpointInfo)
 		}
 	}
 
+	for _, ipAddr := range epInfo.IPsToRouteViaHost {
+		// Get EB rules.
+		log.Printf("[net] Getting EB rules for routing via host")
+		out, err := ebtables.GetEBRules()
+		if err != nil {
+			log.Printf("[net] Failed to get EB rules: %v", err)
+			return err
+		}
+
+		log.Printf("[net] Checking if EB rule already exists for IP address %v", ipAddr)
```

Comment:
move this to generic function in ebtable package. can be

EbtableRuleExists(tableName, chainName, matchSet string). You can call getebtablerules function inside this call